packages/relay/src/routes/post.ts

```diff
@@ -20,167 +15,54 @@ export default (
-            await fetchSinglePost(req, res, db)
-        })
-    )
-}
-
-async function fetchPosts(req, res, db: DB) {
-    try {
-        if (req.query.query === undefined) {
-            const posts = await db.findMany('Post', {
-                where: {
-                    status: 1,
-                },
-            })
-            res.json(posts)
-            return
-        }
-
-        const epks = req.query.epks ? req.query.epks.split('_') : undefined
-
-        const posts = await db.findMany('Post', {
-            where: {
-                epochKey: epks,
-            },
-        })
-
-        res.json(posts.slice(0, Math.min(LOAD_POST_COUNT, posts.length)))
-    } catch (error: any) {
-        res.status(500).json({ error })
-    }
-}
-
-async function createPost(
-    req,
-    res,
-    db: DB,
-    synchronizer: UnirepSocialSynchronizer,
-    helia: Helia
-) {
-    try {
-        const { content, publicSignals, proof } = req.body
-
-        // verify epochKeyProof of user
-        const epochKeyProof = new EpochKeyProof(
-            publicSignals,
-            proof,
-            synchronizer.prover
-        )
-
-        // get current epoch and unirep contract
-        const epoch = await synchronizer.loadCurrentEpoch()
-
-        // check if epoch is valid
-        const isEpochvalid = epochKeyProof.epoch.toString() === epoch.toString()
-        if (!isEpochvalid) {
-            res.status(400).json({ error: 'Invalid Epoch' })
-            return
-        }
-
-        // check if state tree exists in current epoch
-        const isStateTreeValid = await synchronizer.stateTreeRootExists(
-            epochKeyProof.stateTreeRoot,
-            Number(epochKeyProof.epoch),
-            epochKeyProof.attesterId
-        )
-        if (!isStateTreeValid) {
-            res.status(400).json({ error: 'Invalid State Tree' })
-            return
-        }
-
-        // check if proof is valid
-        const isProofValid = await epochKeyProof.verify()
-        if (!isProofValid) {
-            res.status(400).json({ error: 'Invalid proof' })
-            return
-        }
-
-        const appContract = new ethers.Contract(APP_ADDRESS, ABI)
-
-        // post content
-        let calldata: any
-        let cid: any
-        if (content) {
-            // if the content is not empty, post the content
-            calldata = appContract.interface.encodeFunctionData('post', [
-                epochKeyProof.publicSignals,
-                epochKeyProof.proof,
-                content,
-            ])
-
-            // store content into helia ipfs node with json plain
-            const { json } = await eval("import('@helia/json')")
-            const heliaJson = json(helia)
-            const IPFSContent = {
-                content: content,
+            const id = req.params.id
+            if (!id) {
+                console.log('id is undefined')
+                res.status(400).json({ error: 'id is undefined' })
```

Comment:
should add return to finish this request
or should throw error like comment api